src/views/splash/presentation.jsx

```diff
@@ -262,11 +400,13 @@ var SplashPresentation = injectIntl(React.createClass({
                         Object.keys(this.props.user).length !== 0 ? [
                             <div key="header" className="splash-header">
                                 {this.props.shouldShowWelcome ? [
-                                    <Welcome key="welcome"
```

Comment:
This is based on develop with the HoC banners merged (open the previous lines to see the <TopBanner../> [HoC banners were merge on develop to allow QA] So this can't get deployed before HoC, or we need to undo the HoC merge.